packages/perseus/src/components/svg-image.tsx

```diff
@@ -483,24 +476,15 @@ class SvgImage extends React.Component<Props, State> {
                 imageProps.onClick = this._handleZoomClick;
 
                 return (
-                    <FixedToResponsive
-                        className={wrapperClasses}
-                        width={width}
-                        height={height}
-                        constrainHeight={this.props.constrainHeight}
-                        allowFullBleed={
-                            this.props.allowFullBleed &&
-                            isImageProbablyPhotograph(imageSrc)
-                        }
-                    >
```

Comment:
Ideally, I'd completely remove this `FixedToResponsive` component altogether, since modern CSS/React should be able to handle responsive sizing. However, weird things happen to the `Video` component if I remove this, so I'm leaving it for now.